test/files/refresh.js

```diff
@@ -150,7 +150,11 @@ const videos = [
     type: 'game',
     basicInfo: true,
     skip: ['get_video_info'],
-  }
+  },
+  {
+    id: '3IqtmUscE_U',
+    type: 'related',
+  },
```

Comment:
can you add `skip: ['get_video_info']` and maybe `'player'` in there too, since we mainly care about just the `watch.html` page

Reply:
also add `'player'` here so that the html5player file doesn't get saved

Reply:
oh wait it's not as simple as that, `skip` only matches exact filenames, and the player changes all the time. might change that later

Reply:
Ok so .. I shouldn't add it right ? Since it wont work for now

Reply:
yeah it's fine for now. I'll add it in later